Makefile

```diff
@@ -0,0 +1,69 @@
+NAME    := daos
```

Comment:
I am really not keen on adding a makefile at the root of the repo. Could we put all the RPM stuff under something like utils/rpms?
Also, why not using scons?

Reply:
Putting the `Makefile` at the root of the repo was just to make it consistent with how we build RPMs for the other dozen or so projects.  It can be moved for this project.

Using `scons` is definitely a target, but trying to implement it up front would have delayed getting RPMs available by their due date due to needing to figure out how to create an RPM _builder_ in `scons`.  RPM building has "interesting" dependency relationship deriving problems.  It's not something I don't think `scons` can handle.  On the contrary, I think `scons` is probably a much better tool than `make` for it.  It just needs some time and prototyping to figure it all out.

Reply:
so the make file just builds the RPM assuming scons has already run?

Reply:
The `Makefile` runs `rpmbuild` which just follows the recipe in the `daos.spec`.  The `daos.spec` recipe actually runs `scons` to build daos and then packages up that build product into the RPM.

It is a goal of course to replace the `Makefile` with a proper scons RPM builder at some point.